[2025-04-09T15:44:00.430+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T15:44:00.454+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-04-09T15:43:36.468087+00:00 [queued]>
[2025-04-09T15:44:00.462+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-04-09T15:43:36.468087+00:00 [queued]>
[2025-04-09T15:44:00.462+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-09T15:44:00.607+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_trusted> on 2025-04-09 15:43:36.468087+00:00
[2025-04-09T15:44:00.612+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=597) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-09T15:44:00.614+0000] {standard_task_runner.py:72} INFO - Started process 599 to run task
[2025-04-09T15:44:00.614+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_trusted', 'manual__2025-04-09T15:43:36.468087+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmp23ahsqkb']
[2025-04-09T15:44:00.615+0000] {standard_task_runner.py:105} INFO - Job 86: Subtask run_dbt_trusted
[2025-04-09T15:44:00.717+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-04-09T15:43:36.468087+00:00 [running]> on host 5a8aeda502d0
[2025-04-09T15:44:00.857+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_trusted' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T15:43:36.468087+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T15:43:36.468087+00:00'
[2025-04-09T15:44:00.858+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-09T15:44:00.875+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T15:44:00.881+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:trusted']
[2025-04-09T15:44:00.890+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T15:44:00.892+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-04-09T15:44:00.892+0000] {subprocess.py:93} INFO - + dbt run --select tag:trusted
[2025-04-09T15:44:03.189+0000] {subprocess.py:93} INFO - [0m15:44:03  Running with dbt=1.8.0
[2025-04-09T15:44:03.509+0000] {subprocess.py:93} INFO - [0m15:44:03  Registered adapter: postgres=1.8.2
[2025-04-09T15:44:04.055+0000] {subprocess.py:93} INFO - [0m15:44:04  Found 9 models, 10 sources, 428 macros
[2025-04-09T15:44:04.059+0000] {subprocess.py:93} INFO - [0m15:44:04
[2025-04-09T15:44:04.217+0000] {subprocess.py:93} INFO - [0m15:44:04  Concurrency: 1 threads (target='dev')
[2025-04-09T15:44:04.217+0000] {subprocess.py:93} INFO - [0m15:44:04
[2025-04-09T15:44:04.228+0000] {subprocess.py:93} INFO - [0m15:44:04  1 of 4 START sql table model driven_trusted.non_pii_data ....................... [RUN]
[2025-04-09T15:44:07.225+0000] {subprocess.py:93} INFO - [0m15:44:07  1 of 4 OK created sql table model driven_trusted.non_pii_data .................. [[32mSELECT 137378[0m in 2.99s]
[2025-04-09T15:44:07.229+0000] {subprocess.py:93} INFO - [0m15:44:07  2 of 4 START sql table model driven_trusted.payment_data ....................... [RUN]
[2025-04-09T15:44:08.982+0000] {subprocess.py:93} INFO - [0m15:44:08  2 of 4 OK created sql table model driven_trusted.payment_data .................. [[32mSELECT 137378[0m in 1.75s]
[2025-04-09T15:44:08.986+0000] {subprocess.py:93} INFO - [0m15:44:08  3 of 4 START sql table model driven_trusted.pii_data ........................... [RUN]
[2025-04-09T15:44:11.107+0000] {subprocess.py:93} INFO - [0m15:44:11  3 of 4 OK created sql table model driven_trusted.pii_data ...................... [[32mSELECT 137378[0m in 2.12s]
[2025-04-09T15:44:11.111+0000] {subprocess.py:93} INFO - [0m15:44:11  4 of 4 START sql table model driven_trusted.technical_data ..................... [RUN]
[2025-04-09T15:44:12.483+0000] {subprocess.py:93} INFO - [0m15:44:12  4 of 4 OK created sql table model driven_trusted.technical_data ................ [[32mSELECT 137378[0m in 1.37s]
[2025-04-09T15:44:12.500+0000] {subprocess.py:93} INFO - [0m15:44:12
[2025-04-09T15:44:12.501+0000] {subprocess.py:93} INFO - [0m15:44:12  Finished running 4 table models in 0 hours 0 minutes and 8.44 seconds (8.44s).
[2025-04-09T15:44:12.568+0000] {subprocess.py:93} INFO - [0m15:44:12
[2025-04-09T15:44:12.569+0000] {subprocess.py:93} INFO - [0m15:44:12  [32mCompleted successfully[0m
[2025-04-09T15:44:12.570+0000] {subprocess.py:93} INFO - [0m15:44:12
[2025-04-09T15:44:12.571+0000] {subprocess.py:93} INFO - [0m15:44:12  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-04-09T15:44:13.504+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T15:44:13.569+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-09T15:44:13.570+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=run_dbt_trusted, run_id=manual__2025-04-09T15:43:36.468087+00:00, execution_date=20250409T154336, start_date=20250409T154400, end_date=20250409T154413
[2025-04-09T15:44:13.655+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T15:44:13.668+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-09T15:44:13.699+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
